test/eofparse/eofparse.cpp

```diff
@@ -52,22 +67,15 @@ int main()
             }
 
             const auto& eof = *o;
-            const auto err = evmone::validate_eof(EVMC_PRAGUE, eof);
+            const auto err = evmone::validate_eof(rev, eof);
             if (err != evmone::EOFValidationError::success)
             {
                 std::cout << "err: " << evmone::get_error_message(err) << "\n";
                 continue;
             }
 
             const auto header = evmone::read_valid_eof1_header(eof);
-            std::cout << "OK ";
```

Comment:
I believe this format was used by some cross-client fuzzing, maybe we should better change retesteth to accept it.